README.md

```diff
@@ -1 +1,63 @@
-# FunnySectors
\ No newline at end of file
+
+# Funny Sectors
+
+A lightweight plugin for managing large minecraft networks
+
+
+## Important note
+
+We do not have a working demo yet. [we](https://github.com/FunnyGuilds/FunnySectors/graphs/contributors) are actively working on creating one.
```

Comment:
Do we really need to state it?

Reply:
Well... let's say someone is looking thru FunnyGuilds organization and finds our project. He might be confused - no info about how to use it, no CI link. I do think that stating that is important. You can always change readme later. (Let's say with first working demo or first beta release)